### PR TITLE
GCP CloudRun NIM example L4 llama3-8b-instruct:1.0.0

### PR DESCRIPTION
**Pull request by dfisk**
In support of Cloud Run GPU support launch

### CODE REVIEW

File: cloud-service-providers/google-cloud/cloudrun/env
Comment (abhisheksawarkar) [inline cloud-service-providers/google-cloud/cloudrun/env:1]: Please remove your credentials and IDs

File: cloud-service-providers/google-cloud/cloudrun/test/env
Comment (abhisheksawarkar) [inline cloud-service-providers/google-cloud/cloudrun/test/env:1]: Please remove the credentials and IDs

File: cloud-service-providers/google-cloud/cloudrun/README
Comment (FortunaZhang) [inline cloud-service-providers/google-cloud/cloudrun/README:1]: Is it in compliance to provide real GCP project name and id in public git repo?

File: cloud-service-providers/google-cloud/cloudrun/test/Dockerfile_build [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:22.04
+EXPOSE 3333
+RUN apt-get update && apt-get install curl -y
+RUN apt-get update && apt-get install python3-pip -y
+RUN pip install fastapi
+RUN pip install "uvicorn[standard]"
+COPY source/query.sh /usr/local/bin/query.sh
+COPY source/http_respond.py  /home/nemo/http_respond.py
+RUN chmod +x /usr/local/bin/query.sh
+ENV CLOUD_RUN_ENDPOINT_URL=https://llama-3-8b-instruct-kcifu7ad4a-uc.a.run.app
```
Comment (FortunaZhang) [inline cloud-service-providers/google-cloud/cloudrun/test/Dockerfile_build:10]: Could I know the source of ENDPOINT URL, and if it has public access?

File: cloud-service-providers/google-cloud/cloudrun/test/README
Comment (FortunaZhang) [inline cloud-service-providers/google-cloud/cloudrun/test/README:1]: It looks like test files are run NIM with CPU, how could user better use scripts in /test to help GPU deployment?

File: cloud-service-providers/google-cloud/cloudrun/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+FROM nvcr.io/nim/meta/llama3-8b-instruct:1.0.0
+MAINTAINER "Dave Fisk" dfisk@nvidia.com
```
Comment (supertetelman) [inline cloud-service-providers/google-cloud/cloudrun/Dockerfile:2]: We don't need a MAINTAINER email here

File: cloud-service-providers/google-cloud/cloudrun/README [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Create an env file to place all exported environment variables.
+Here is a complete example:
+$ cat env
+export SERVICE_ACCOUNT_ID=dfisk-nemoms-vertex-ai-study
+export PROJECTID=k80-exploration
+export PROJECTUSER=dfisk
+export PROJECTNUM=342959614509
+export REGION=us-central1
+export GCSBUCKET=dfisk-model-data
```
Comment (supertetelman) [inline cloud-service-providers/google-cloud/cloudrun/README:29]: remove

File: cloud-service-providers/google-cloud/cloudrun/README [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Create a file with your NGC token in source/ngc-token, then
+create a secret from your NGC token for use by the NIM:
+$ echo -n $(cat source/ngc-token) | gcloud secrets create nim-ngc-token \
+    --replication-policy="automatic" \
+    --data-file=-
+
+Create an env file to place all exported environment variables.
+Here is a complete example:
+$ cat env
+export SERVICE_ACCOUNT_ID=dfisk-nemoms-vertex-ai-study
```
Comment (supertetelman) [inline cloud-service-providers/google-cloud/cloudrun/README:24]: remove all pii

File: cloud-service-providers/google-cloud/cloudrun/env [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export SERVICE_ACCOUNT_ID=dfisk-nemoms-vertex-ai-study
```
Comment (supertetelman) [inline cloud-service-providers/google-cloud/cloudrun/env:1]: Remove all pii

**Comment by supertetelman** [comment]
Merged through in https://github.com/NVIDIA/nim-deploy/pull/74

**Pull request closed without merging**